english/eating-walls/find-a-toilet/relieve/relieve-yourself.md

```diff
@@ -1,3 +1,7 @@
 Gross!
 
 You turn and run away in shame.
+
+But almost immediately you slam straight into a pole and break your front teeth.  You cry out in agony and turn toward a faint light coming from a door you hadn't seen before.  When you reach the door, you see it is a dental office.  How convenient.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.